styleguide/components/41-organisms/contact-box/contact-box.twig

```diff
@@ -29,39 +29,69 @@
                 'modifier': 'contact-email'
               } %}
             </address>
+              {% if contact.opening_hours|length > 0 %}
+                <span class="opening-hours open">
+                  {{ 'Today open - ' }}<time>{{ contact.opening_hours }}</time>
+                </span>
+              {% else %}
+              <span class="opening-hours closed">
+                {{ 'Closed today' }}
+              </span>
+              {% endif %}
           </li>
         {% endfor %}
       </ul>
+      <h3>
+        <button class="accordion--button"
+                aria-expanded="false"
+                aria-controls="contact-box--content">
+          {{ 'There are 5 other contacts available' }}
+        </button>
+      </h3>
+
+      <div class="accordion--content"
+        id="contact-box--content">
```

Comment:
is de id noodzakelijk, kans is klein dat we deze uniek kunnen houden.

Reply:
Heb unique id's toegevoegd in de stijlgids. De drupal templates krijgen ook een unique id nu.